boards/st/sensortile_box_pro/Kconfig.defconfig

```diff
@@ -20,46 +20,7 @@ config SPI_STM32_INTERRUPT
 	default y
 	depends on SPI
 
-if BOARD_SERIAL_BACKEND_CDC_ACM
-
-config USB_DEVICE_STACK
-	default y
-
-config USB_CDC_ACM
-	default SERIAL
-
-config USB_DEVICE_INITIALIZE_AT_BOOT
-	default y if CONSOLE
-
-config SHELL_BACKEND_SERIAL_CHECK_DTR
-	default SHELL
-	depends on UART_LINE_CTRL
-
-config UART_LINE_CTRL
-	default SHELL
-
-config USB_DEVICE_REMOTE_WAKEUP
-	default n
-
-config USB_DEVICE_VID
-	default 0x0483
-
-config USB_DEVICE_PID
-	default 0x5740
-
-config USB_DEVICE_PRODUCT
-	default "Zephyr CDC SensorTile.box PRO"
-
-if LOG
-
-# Logger cannot use itself to log
-choice USB_CDC_ACM_LOG_LEVEL_CHOICE
-	default USB_CDC_ACM_LOG_LEVEL_OFF
-endchoice
-
-endif # LOG
-
-endif # BOARD_SERIAL_BACKEND_CDC_ACM
+source "${ZEPHYR_BASE}/boards/common/Kconfig.cdc_acm_serial"
 
 DT_CHOSEN_ZEPHYR_CONSOLE := zephyr,console
```

Comment:
Shouldn't this be cleaned up as well ?

Reply:
This and following lines to be cleaned up as now conflicts with configuration provided in `${ZEPHYR_BASE}/boards/common/usb/Kconfig.cdc_acm_serial`